notebooks/01-lightcurves/01-Lightcurves.ipynb

```diff
@@ -56,9 +56,11 @@
    "source": [
```

Comment:
<p>ok, last round of feedback. these items are relatively minor and then we should be good to go!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>great!</p>